examples_tests/31.SkinningDataBenchmark/5.vert

```diff
@@ -0,0 +1,126 @@
+#include "common.glsl"
+
+#extension GL_KHR_shader_subgroup_ballot : require
+
+struct BoneData
+{
+    mat4 boneMatrix;
+    mat4x3 normalMatrix;
+};
+
+layout(std430, set = 0, binding = 0, row_major) readonly buffer BoneMatrices_struct
+{
+    BoneData data[];
+} boneSSBO_structs;
+layout(std430, set = 0, binding = 1) readonly buffer BoneMatrices_dword
+{
+    uint data[];
+} boneSSBO_dwords;
+
+#ifndef BENCHMARK
+layout(location = 0) in vec3 pos;
+layout(location = 3) in vec3 normal;
+layout(location = 0) out vec3 vNormal;
+#endif
+layout(location = 4) in uint boneID;
+
+#define OBJ_DWORDS 32 // sizeof(BoneData), must be PoT
+struct BoneData_dword
+{
+    uint data[OBJ_DWORDS];
+};
+BoneData toBoneData(in BoneData_dword bone)
+{
+    BoneData retval;
+    //tpose because it was loaded as row_major
+    //one-liner because glslang doesnt support multiline preproc definitions
+#define GET_BONE_MATRIX_COL(c) retval.boneMatrix[c].x = uintBitsToFloat(bone.data[c]);retval.boneMatrix[c].y = uintBitsToFloat(bone.data[c+4]);retval.boneMatrix[c].z = uintBitsToFloat(bone.data[c+8]);retval.boneMatrix[c].w = uintBitsToFloat(bone.data[c+12])
+    
+    GET_BONE_MATRIX_COL(0);
+    GET_BONE_MATRIX_COL(1);
+    GET_BONE_MATRIX_COL(2);
+    GET_BONE_MATRIX_COL(3);
+
+    //tpose because it was loaded as row_major
+    //one-liner because glslang doesnt support multiline preproc definitions
+#define GET_NORMAL_MATRIX_COL(c) retval.normalMatrix[c].x = uintBitsToFloat(bone.data[16+c]);retval.normalMatrix[c].y = uintBitsToFloat(bone.data[16+c+4]);retval.normalMatrix[c].z = uintBitsToFloat(bone.data[16+c+8])
+
+    GET_NORMAL_MATRIX_COL(0);
+    GET_NORMAL_MATRIX_COL(1);
+    GET_NORMAL_MATRIX_COL(2);
+    GET_NORMAL_MATRIX_COL(3);
+
+    return retval;
+}
+#define COALESCING_DWORDS_LOG2 4 // GCN can fetch only 64bytes in a single request
+#define SUBGROUP_THRESH 16
```

Comment:
change the theshold to `(0x1<<COALESCING_DWORDS_LOG2)`

Also does changing the `COALESCING_DWORDS_LOG2` to 5 and deleting the `notEnoughInvocations` (always false) change the perf of the benchmark much?